app/docker/views/containers/create/createContainerController.js

```diff
@@ -619,62 +581,179 @@ function ($q, $scope, $state, $timeout, $transition$, $filter, Container, Contai
     return true;
   }
 
-  $scope.create = function () {
-    confirmCreateContainer()
-    .then(function success(confirm) {
-      if (!confirm) {
-        return false;
+
+  function create() {
+    var oldContainer = null;
+
+    $scope.state.actionInProgress = true;
```

Comment:
The `actionInProgress` flag should be set to true only after the confirmation modal has been clicked (if we replace a container).